modules/ztp-image-based-upgrade-prep-label-extramanifests.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+// * edge_computing/image-based-upgrade/cnf-preparing-for-image-based-upgrade.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrade-labeling-extramanifests-with-ztp_{context}"]
+= Labeling extra manifests for the image-based upgrade with GitOps ZTP
+
+The {lcao} only extracts extra manifests that are labeled with the `lca.openshift.io/target-ocp-version: <target_version>` label.
+
+.Prerequisites
+
+* Provision one or more managed clusters with GitOps ZTP.
+* Log in as a user with `cluster-admin` privileges.
+* Generate a seed image from a compatible seed cluster.
+* Create a separate partition on the target cluster for the container images that is shared between stateroots. For more information about, see _Additional resources_.
+* Deploy a version of {lcao} that is compatible with the version used with the seed image.
+
+.Procedure
+
+. If you have manifests that you want to be extracted and applied during the upgrade, label them with the `lca.openshift.io/target-ocp-version: <target_version>` label in your existing `PolicyGenTemplate` CR:
```

Comment:
Is this an optional step, or is it something I'm always doing if I'm "Labeling extra manifests for the image-based upgrade with GitOps ZTP." Unsure about the "If" here.

Reply:
Good point, removed if and rephrased a tiny bit:
> Label your required extra manifests with...